js/script.js

```diff
@@ -112,8 +120,28 @@ document.addEventListener("DOMContentLoaded", () => {
             toggleUIConnected(false);
         });
     });
+    butOffline.addEventListener("click", () => {
+        clickOffline();/*.catch(async (e) => {
+            // Default Help Message:
+            // if we've failed to catch the message before now, we need to give
+            // the generic advice: reconnect, refresh, go to support
+            errorMsg(
+                `Connection Error, your board may be incompatible. Things to try:\n` +
+                `1. Reset your board and try again.\n` +
+                `  - Look for a little black button near the power port.\n` +
+                `2. Refresh your browser and try again.\n` +
+                `3. Make sure you are not connected in another browser tab.\n` +
+                `4. Double-check your board type and serial port selection.\n` +
+                `5. Post on the Support Forum (link above) with this info:\n\n` +
+                `"Firmware Tool: ${e}"\n`
+            );
+            await disconnect();
+            toggleUIConnected(false);
+        });*/
```

Comment:
what's this commented code hanging around for?

Reply:
I was debugging. I need to uncomment.